boards/nxp/frdm_kl25z/Kconfig.frdm_kl25z

```diff
@@ -4,3 +4,4 @@
 config BOARD_FRDM_KL25Z
 	select SOC_MKL25Z4
 	select SOC_PART_NUMBER_MKL25Z128VLK4
+	select PINCTRL
```

Comment:
this does not go here

Reply:
Where does this belong if the board is using the pinctrl api?

Reply:
Where is the .c file in the board folder where pinctrl is used?

Reply:
Ah, I apologize for the confusion, somehow I thought I saw a pinctrl API in this directory, but there is one for the twr board so the question still applies, the select for this board though I can remove on the next update.